### ISSUE TITLE
Jewels unobtainable

### ISSUE THREAD

**Issue by solvedDev** [solvedDev/astroExp#4]
As the machine blocks have been removed, the jewels are currently unobtainable.

**Comment by AlexCreeper25** [comment]
With the new way of getting gems, you could close this issue

**Comment by solvedDev** [comment]
You're right, I am getting there but it's not a 100% done yet. E.g.: I still need to think about how I want to make dark jewels achievable. Additionally, I need to reintroduce the corresponding villager trades to turn shards into the full jewel counterpart.
